envs/docker-common.yml

```diff
@@ -2,30 +2,33 @@ version: '2.1'
 
 services:
   postgres:
-    image: postgres:9.6.2
+    image: postgres:15.2
     environment:
       - POSTGRES_PASSWORD=my-secret-password
       - POSTGRES_USER=smart
       - POSTGRES_DB=smart
+      - POSTGRES_HOST=postgres
+      - POSTGRES_HOST_AUTH_METHOD=md5
     ports:
       - "${EXTERNAL_POSTGRES_PORT:-5432}:5432"
     volumes:
-      - smart_pgdata:/var/lib/postgresql/data
+      - smart_pgdata_15_2:/var/lib/postgresql/data
```

Comment:
Getting this error when I try any docker-compose command from the envs/dev folder

`service "postgres" refers to undefined volume smart_pgdata_15_2: invalid compose project`

Reply:
When I dropped the _15_2 change it built fine.

Reply:
Fixed. The change to the volume name was driven by the prod migration, where I needed to support multiple pg volumes at once.